src/services/orderbook_service.ts

```diff
@@ -173,16 +174,21 @@ export class OrderBookService {
         const paginatedApiOrders = paginationUtils.paginate(apiOrders, page, perPage);
         return paginatedApiOrders;
     }
-    constructor(connection: Connection, meshClient?: WSClient) {
+    constructor(connection: Connection, meshClient?: WSClient, meshHttpEndpoint?: string) {
         this._meshClient = meshClient;
+        this._meshHttpEndpoint = meshHttpEndpoint;
         this._connection = connection;
     }
     public async addOrderAsync(signedOrder: SignedOrder): Promise<void> {
         return this.addOrdersAsync([signedOrder]);
     }
     public async addOrdersAsync(signedOrders: SignedOrder[]): Promise<void> {
-        if (this._meshClient) {
-            const { rejected } = await this._meshClient.addOrdersAsync(signedOrders as any);
+        if (this._meshClient && this._meshHttpEndpoint) {
+            const { rejected } = await meshUtils.addOrdersToMeshAsync(
+                this._meshHttpEndpoint,
+                this._meshClient,
+                signedOrders,
+            );
```

Comment:
We should probably still add orders if we don't have the `meshHttpEndpoint` ENV var. This will also save us the headache of updating all launch-kit related things. Alternatively we can pin the version of 0x-api used in launch-kit.

Reply:
@fabioberger suggested adding all orders over HTTP, which I think makes sense, but that means requiring this env var.